srcpkgs/R-cran-tibble/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'R-cran-tibble'
 pkgname=R-cran-tibble
-version=2.1.3
+version=3.0.1
 revision=1
 build_style=R-cran
-makedepends="R-cran-cli R-cran-crayon R-cran-pillar R-cran-rlang R-cran-pkgconfig"
-depends="R-cran-cli R-cran-crayon R-cran-pillar>=1.1.0 R-cran-rlang R-cran-pkgconfig"
+makedepends="R-cran-cli R-cran-crayon R-cran-ellipsis R-cran-fansi R-cran-lifecycle R-cran-magrittr R-cran-pillar R-cran-rlang R-cran-pkgconfig R-cran-vctrs"
+depends="R-cran-cli R-cran-crayon R-cran-ellipsis R-cran-fansi R-cran-lifecycle R-cran-magrittr R-cran-pillar>=1.4.3 R-cran-rlang R-cran-pkgconfig R-cran-vctrs"
```

Comment:
please keep lines at about 80 characters

Reply:
Fixed. I'll keep that in mind for next time thanks.